packages/server/src/utilities/workerRequests.ts

```diff
@@ -65,15 +65,35 @@ async function checkResponse(
 }
 
 // have to pass in the tenant ID as this could be coming from an automation
-export async function sendSmtpEmail(
-  to: string,
-  from: string,
-  subject: string,
-  contents: string,
-  cc: string,
-  bcc: string,
+export async function sendSmtpEmail({
+  to,
+  from,
+  subject,
+  contents,
+  cc,
+  bcc,
+  automation,
+  addInvite,
```

Comment:
could we compress the invitation options into a single object? Right now its not very obvious which options apply to the invitation.

Reply:
compressed into invite object